src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post">
+
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+          class="form_group"
+          type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+        >
+        <br>
+        <label for="name">Name:</label>
+        <input
+          class="form_group"
+          type="text"
+          id="name"
+          name="name"
+          autocomplete="off"
+        >
+        <br>
+        <label for="age">How old are You?</label>
+        <input
+          class="form_group"
+          type="number"
+          id="age"
+          name="age"
+          value="12"
+          min="1"
+          max="100"
+        >
+        <br>
+        <label for="date">Full date of birth:</label>
+        <input
+          class="form_group"
+          type="date"
+          id="date"
+          name="date"
+        >
+        <br>
+        <label for="box">I accept the term of the agreement</label>
+        <input
+          type="checkbox"
+          id="box"
+          name="box"
+        >
+        <br>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label for="adress">E-mail:</label>
+        <input
+          type="email"
+          id="adress"
+          class="form_group"
+          name="adress"
+          placeholder="email@example.com"
+        >
+        <br>
+        <label for="secret">Password:</label>
+        <input
+          type="password"
+          id="secret"
+          name="secret"
+          required
+          autocomplete="off"
+          minlength="8"
+          maxlength="15"
+        >
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form_group">
+          <label for="cat">Do you love cats?</label>
+
+          <label for="yes">Yes</label>
+          <input
+            id="yes"
+            type="radio"
+            name="pet"
+            value="yes"
+          >
+          <label for="no">No</label>
+          <input
+            id="no"
+            type="radio"
+            name="pet"
+            value="No"
+          >
```

Comment:
```suggestion
          <label>
            <input
              id="yes"
              type="radio"
              name="pet"
              value="yes"
            >
            Yes
          </label>
          <label>
            <input
              id="no"
              type="radio"
              name="pet"
              value="No"
            >
            No
          </label>
```

 You can wrap it into label tag to fix previous comment